casa/System/AipsrcBool.cc

```diff
@@ -58,34 +47,34 @@ Bool AipsrcValue<Bool>::find(Bool &value, const String &keyword,
 uInt AipsrcValue<Bool>::registerRC(const String &keyword,
 				   const Bool &deflt) {
   std::lock_guard<std::mutex> lock(theirMutex);
-  uInt n = Aipsrc::registerRC(keyword, myp_p.ntlst);
-  myp_p.tlst.resize(n);
-  find ((myp_p.tlst)[n-1], keyword, deflt);
+  uInt n = Aipsrc::registerRC(keyword, ntlst);
+  tlst.resize(n);
```

Comment:
```suggestion
  if (n > tlist.size())
    tlst.resize(n);
```

Reply:
Thank you!! This would have introduced a nasty bug.

It's also a reason why to avoid the casacore::Block ; it has just slightly different behaviour compared to std::vector, which can be confusing...